tools/gatk/filter_mutect.nf

```diff
@@ -5,7 +5,7 @@ process FILTERMUTECT {
 
```

Comment:
Eventually I would like to have the memory for each tool specified in the process, yes! 
I have been waiting until I get more data on ROCs and TCGA sample processing because I've been narrowing down the requirements for each tool and was planning on doing all of them in one go